README.md

```diff
@@ -136,6 +137,12 @@ for panoramas, virtual tours, photoshperes, and other 3d zoomable images.
 dezoomify-rs supports downloading individual image planes from such images.
 You need to provide the xml meta-information file for the image.
 
+### NYPLImage
+
+[NYPLIMage](https://digitalcollections.nypl.org) is just a tile-set of png-image
```

Comment:
Maybe let the user know that they can give either the image viewer URL or the config.js URL to dezoomify-rs.